src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5015,7 +5072,15 @@ private static BoundConversion GetConversionIfApplicable(BoundExpression convers
                     {
                         var delegateType = targetType.GetDelegateType();
                         var variableState = GetVariableState(stateForLambda);
-                        VisitLambda(lambda, delegateType, Diagnostics, variableState);
+                        LambdaSymbol updatedLambdaSymbol = null;
+                        if (delegateType is object)
+                        {
+                            var containingSymbol = GetUpdatedContainingSymbol(lambda.Symbol);
+                            updatedLambdaSymbol = lambda.CreateLambdaSymbol(delegateType, containingSymbol);
```

Comment:
>updatedLambdaSymbol = lambda.CreateLambdaSymbol(delegateType, containingSymbol); [](start = 28, length = 80)

Same comment as for VisitLocalDeclaration

Reply:
No longer applies as this code isn't changed.

---
In reply to: [327918732](https://github.com/dotnet/roslyn/pull/38769#discussion_r327918732) [](ancestors = 327918732)